src/naming.jl

```diff
@@ -283,10 +284,18 @@ Same as [`@dict`](@ref) but the key type is `String`.
 """
 macro strdict(vars...)
```

Comment:
So my understanding now is that this works only for `@strdict`? Then it should also be done in `@dict, @ntuple, @savename`.

Reply:
:+1: 
(yes, as I edited above) 
Will do, when I get the chance